src/models/DefaultData.js

```diff
@@ -212,6 +220,70 @@ class DefaultData {
     return links.concat(this.getWorkflowLinks());
   }
 
+  /**
+   * Get the value of an attribute.
+   * @param {ComponentAttribute} attribute - Attribute to get value.
+   * @returns {string | string[]} Value of attribute.
+   */
+
+  getAttributeValue(attribute) {
+    if (attribute.isVariable) {
+      return this.getVariableValue(attribute.value);
+    }
+
+    return attribute.value;
+  }
+
+  /**
+   * Get the ID of the linked component.
+   * @param {ComponentAttribute} attribute - Link to get value.
+   * @returns {string[]} ID of the linked component.
+   */
+  getLinkedComponentsIds(attribute) {
+    const value = this.getAttributeValue(attribute);
+
+    if (value === null) {
+      return [];
+    }
+    if (Array.isArray(value)) {
+      return value;
+    }
+
+    return [value];
```

Comment:
return [this.getComponentIdFromValue(value)];

Reply:
Done.